diff --git a/networkbid/realtime-bidding.proto b/networkbid/realtime-bidding.proto
index 3a62b34..9c330ab 100644
--- a/networkbid/realtime-bidding.proto
+++ b/networkbid/realtime-bidding.proto
@@ -1,5 +1,8 @@
 // Protocol version: v.181
 // Copyright 2020 Google Inc. All Rights Reserved.
+syntax = "proto2";
+option go_package = ".;networkbid";
+package com.google.protos.adx;
 
 // This is the message that Google uses to request bids. A BidRequest
 // includes the ad slot from a single impression.
diff --git a/openrtbadx/openrtb-adx.proto b/openrtbadx/openrtb-adx.proto
index 123bddc..32bc9e9 100644
--- a/openrtbadx/openrtb-adx.proto
+++ b/openrtbadx/openrtb-adx.proto
@@ -1,5 +1,7 @@
 // Protocol version: v.52
+syntax = "proto2";
 import "openrtb.proto";
+option go_package = ".;openrtbadx";
 option java_outer_classname = "AdxExt";
 package com.google.doubleclick;
 
diff --git a/openrtbadx/openrtb.proto b/openrtbadx/openrtb.proto
index 77016fc..791b057 100644
--- a/openrtbadx/openrtb.proto
+++ b/openrtbadx/openrtb.proto
@@ -1,3 +1,5 @@
+syntax = "proto2";
+option go_package = ".;openrtbadx";
 option java_outer_classname = "OpenRtb";
 package com.google.openrtb;
 
